src/platform_impl/macos/window_delegate.rs

```diff
@@ -286,7 +278,6 @@ extern "C" fn window_will_close(this: &Object, _: Sel, _: id) {
 extern "C" fn window_did_resize(this: &Object, _: Sel, _: id) {
     trace_scope!("windowDidResize:");
     with_state(this, |state| {
-        state.emit_resize_event();
         state.emit_move_event();
```

Comment:
Could you add a comment that resize size is reported in `frameDidResize`.

Reply:
done